datadog_checks_base/requirements.in

```diff
@@ -24,7 +24,7 @@ python-dateutil==2.8.1
 pywin32==228; sys_platform == 'win32' and python_version < '3.0'
 pywin32==300; sys_platform == 'win32' and python_version > '3.0'
 pyyaml==5.4.1
-requests==2.25.1
+requests==2.22.0
```

Comment:
Why this downgrade?  I don't see anything about it in the commit message or the PR description.

Reply:
I was trying to rule out whether the requests upgrade may have been causing socket timeout errors in the CI (it wasn't), but I've since reverted this change.